string mode
float64 x_sp
float64 y_sp
float64 z_sp
float64 vx_sp
float64 vy_sp
float64 vz_sp
float64 yaw_sp
